spark/src/test/scala/org/apache/spark/sql/delta/InCommitTimestampSuite.scala

```diff
@@ -325,7 +328,8 @@ class InCommitTimestampSuite
       val ver0Snapshot = deltaLog.snapshot
       assert(DeltaConfigs.IN_COMMIT_TIMESTAMPS_ENABLED.fromMetaData(ver0Snapshot.metadata))
       // Disable ICT in version 1.
-      spark.sql(s"ALTER TABLE delta.`${tempDir.getAbsolutePath}`" +
+      spark.sql(
+        s"ALTER TABLE delta.`${tempDir.getAbsolutePath}`" +
```

Comment:
nit: Missing space between the two string segments? But I guess the SQL parser didn't care?
```suggestion
        s"ALTER TABLE delta.`${tempDir.getAbsolutePath}` " +
```